### DISCUSSION TITLE
replace ":" with " - " as separator between album and subalbum (compilation)

### DISCUSSION THREAD

**Discussion by Damncold** [beetbox/beets#5128] [Q&A]
Hi,

in a compliation, beets is correcting the separator between Album and "subalbum-name" 
from " - "  to ": "

```
Correcting tags from:
    The Time-Life Treasury Of Christmas II - Christmas Joy
To:
    The Time-Life Treasury of Christmas II: Christmas Joy
```
(i know its not THIS time of the year anymore ...)

Which is not working for Windows.
Is there any way to leave it like it is, or tell replace something like this
`'\s*:\s*':' - '`

Tried for an hour without success :-\

**Comment by JOJ0** [comment]
Are you sure this is a replacement handled by "beets magic" and not just what your tagging source provides (MusicBrainz, Discogs, ...)?

**Comment by JOJ0** [comment]
Anyway, which plugin are you using for the replacments?

There are available:
https://beets.readthedocs.io/en/latest/plugins/advancedrewrite.html
https://beets.readthedocs.io/en/latest/plugins/rewrite.html#rewrite-plugin
https://beets.readthedocs.io/en/latest/plugins/substitute.html
